src/Storages/StorageFile.cpp

```diff
@@ -481,19 +483,31 @@ class StorageFileSource : public ISource
                  std::make_shared<DataTypeLowCardinality>(std::make_shared<DataTypeString>()),
                  "_file"});
 
+        /// proton : starts
+        if (is_streaming_)
+        {
+            for (const auto * reserved_col :
+                 {&ProtonConsts::STREAMING_WINDOW_START,
+                  &ProtonConsts::STREAMING_WINDOW_END,
+                  &ProtonConsts::STREAMING_TIMESTAMP_ALIAS})
+                if (header.has(*reserved_col))
+                    header.erase(*reserved_col);
+        }
+        /// proton : ends
```

Comment:
These columns has been removed in `ProxyStream`. I think we don't need this code.